packages/suite/src/components/suite/graph/TransactionsGraph/GraphTooltipBase.tsx

```diff
@@ -200,8 +194,7 @@ export const GraphTooltipBase = (props: GraphTooltipBaseProps) => {
                     {props.balance && (
                         <Row>
                             <Value>
-                                <Sign $color="transparent">+</Sign>
-                                {props.balance}
+                                <Row $noBottomMargin>{props.balance}</Row>
```

Comment:
![Screenshot 2024-11-27 at 16 20 30](https://github.com/user-attachments/assets/9b660a19-224f-4344-8b28-57119c599772)
🤯

Reply:
Is there some request to remove those signs?